src/sdk/pynni/nni/trial.py

```diff
@@ -60,19 +79,49 @@ def get_current_parameter(tag=None):
     return _params['parameters'][tag]
 
 def get_experiment_id():
+    """
+    Get experiment ID.
+
+    Returns
+    -------
+    str
+        Identifier of current experiment
+    """
     return _experiment_id
 
 def get_trial_id():
+    """
+    Get trial job ID.
+
+    Returns
+    -------
+    str
+        Identifier of current trial job which is calling this API.
+    """
     return _trial_id
 
 def get_sequence_id():
+    """
+    Get trial job sequence nubmer. A sequence number is an integer value assigned to each trial job base on the
```

Comment:
What's the difference between trial id and sequence id?
Can you give an example here?

Reply:
Thanks, example and more details added.